helpers/for.js

```diff
@@ -3,8 +3,10 @@
 const _ = require('lodash');
 
 const factory = () => {
-    return function(from, to, context) {
-        const options = arguments[arguments.length - 1];
+    return function(from, to, context, options) {
+        //function takes variable number of arguments, options is last parameter and also needed for 
```

Comment:
I don't like the fact that we are using two patterns for argument shuffling in this function. There is code below that does something different, including a stricter check on the options object. I feel like you should integrate your options param into that code rather than doing this here.

Reply:
fixed.

Reply:
I think there was weird git behavior that lead to for and if not being changed on last rebase.